Content.Client/GameObjects/Components/Kitchen/ReagentGrinderBoundUserInterface.cs

```diff
@@ -0,0 +1,316 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+using System.Text;
+using Robust.Client.GameObjects.Components.UserInterface;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Shared.GameObjects.Components.UserInterface;
+using Robust.Shared.Localization;
+using Robust.Shared.Maths;
+using Content.Shared.Kitchen;
+using Robust.Shared.GameObjects;
+using Content.Shared.Chemistry;
+using Robust.Shared.IoC;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Prototypes;
+using Robust.Client.Graphics;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.GameObjects.Components.Kitchen
+{
+    public class ReagentGrinderBoundUserInterface : BoundUserInterface
+    {
+        [Dependency] private readonly IEntityManager _entityManager = default!;
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
```

Comment:
Could these 2 just be resolved when needed if they're called infrequently?

Reply:
they are called very frequently. every  time the contents get refreshed